advocacy_docs/community/contributing/styleguide.mdx

```diff
@@ -125,12 +125,18 @@ Enter the following information:
 
 ### Sentence length
 
-Use simple and direct language and keep your sentences short. Avoid combining sentences, which makes the content complicated. Maximum sentence length is 26 words when possible.
+Use simple and direct language and keep your sentences short. Avoid combining sentences, which makes the content complicated. The preferred maximum sentence length is 26 words.
 
 ### Contractions
 
 In keeping with the casual and friendly tone, use contractions. However, use common contractions (isn’t, can’t, don’t). Don't use contractions that are unclear or difficult to pronounce (there’ll).
 
+In new reference material, don't use contractions.
+
+For instructions that concern the safety of a person, equipment, or code, it's okay to spell out the words if you choose to, for example: 
```

Comment:
I am not sure I am using the best examples in my suggestion, but I believe we could be more specific:
```suggestion
For instructions that concern important lifecycle operations of an environment, the functionality of databases, or when warning against actions that could interrupt a procedure, it's okay to spell out the words if you choose to, for example: 
```